code/modules/mob/grab/normal/grab_normal.dm

```diff
@@ -72,26 +76,30 @@
 		return FALSE
 
 	var/mob/living/assailant = G.assailant
-	if(!assailant || !assailant.skill_check(SKILL_COMBAT, SKILL_ADEPT))
+	if(!assailant)
 		return FALSE
 
 	var/obj/item/organ/external/O = G.get_targeted_organ()
 	if(!O)
 		to_chat(assailant, SPAN_WARNING("\The [affecting] is missing that body part!"))
 		return  FALSE
 
-	if(!O.dislocated)
+	if(!assailant.skill_check(SKILL_COMBAT, SKILL_ADEPT))
+		to_chat(assailant, SPAN_WARNING("You attempt clumsily but can't seem to dislocate \the [affecting]'s [O.joint]!"))
```

Comment:
Would suggest:
`You clumsily attempt to dislocate \the [affecting]'s [O.name], but fail!`